docs/user/rollout-policies.md

```diff
@@ -0,0 +1,107 @@
+# Rollout Device Selection and Rollout Disruption Budget
+
+## Overview
+
+When performing a rollout using `flightctl`, it is crucial to manage which devices participate in the rollout and how much disruption is acceptable. This document outlines the device selection process and the concept of the rollout disruption budget to ensure controlled and predictable rollouts.
```

Comment:
"which devices participate in the rollout" makes it sound like not all devices in a fleet need to be rolled out.  In the end, all of them will be.